walkthroughs/tls-with-acm/infrastructure/ecr-repositories.yaml

```diff
@@ -1,19 +1,11 @@
 Parameters:
 
-  GatewayImageName:
-    Description: The name for the gateway image
-    Type: String
-
   ColorTellerImageName:
     Description: The name for the color teller image
     Type: String
 
 
 Resources:
-  GatewayRepository:
-    Type: AWS::ECR::Repository
-    Properties:
-      RepositoryName: !Ref GatewayImageName
```

Comment:
I think I failed to remove this in the `tls-with-file` walk through. Could you go ahead and clean that up in this PR as well?